.github/workflows/nfazure.yml

```diff
@@ -0,0 +1,50 @@
+name: nf-azure CI
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Build nf-nomad
+    if: "!contains(github.event.head_commit.message, '[ci skip]')"
+    runs-on: nfazure
+    timeout-minutes: 10
```

Comment:
Maybe we'll need a longer timeout?

Reply:
I hope 10 mnts will be enough but it will depends how many pipelines we want to include in the test for example ... so we'll see